src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -3,17 +3,28 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 #---------------------------------------------------------------------------------------------
 
+import binascii
+import json
+import os
+import os.path
 import platform
 import random
 import string
+import sys
 from six.moves.urllib.request import urlretrieve #pylint: disable=import-error
+import time
+
+from msrestazure.azure_exceptions import CloudError
 
 import azure.cli.core._logging as _logging
 from azure.cli.command_modules.acs import acs_client, proxy
+from azure.cli.command_modules.vm.mgmt_acs.lib import \
+    AcsCreationClient as ACSClient
```

Comment:
ack.  I promise to fix this asap, but I need to get this in for the 11/11 cut.

Thanks for the flexibility.